### ISSUE TITLE
Category selection is inconsistent

### ISSUE THREAD

**Issue by Deguello** [actualbudget/releases#89]
I haven't made a complete list, though I probably should, but in some places in the (Mac) app, a category can be selected by partially typing the name and hitting TAB once there's only one option and other places require ENTER to make the selection.

IMO, TAB is preferable but consistency is even more preferable.

**Comment by jlongster** [comment]
Yeah, this definitely needs some UX work. I have some concern with tab where it's easy to accidentally select something when you didn't mean to. However I think you're right; tab is the way to go and if you want to tab away without saving you do "escape" first and then tab.
